README.md

```diff
@@ -35,6 +37,9 @@ grpc.xpring.tech:80
 Xpring is working on building a zero-config way for XRP node users to deploy and use the adapter as an open-source component of [rippled](https://github.com/ripple/rippled). Watch this space!
 
 ## Usage
+
+*Note*: Xpring SDK only works with X-Addresses. You can learn more about working with X-Addresses in the `Utils` section below and at http://xrpaddress.info.
```

Comment:
```suggestion
 **Note:** Xpring SDK only works with the X-Address format. For more information about this format, see the [Utilities section](#utilities) and <http://xrpaddress.info>.
```

Reply:
My inclination is to add a link to this page, too: https://xrpl.org/accounts.html 

But that seems like it would probably be TMI for devs who just want to use this SDK.

Reply:
Done.